### PR TITLE
Rename "sourcePath" in swingset configs to "sourceSpec"

### PR DESCRIPTION
**Pull request by FUDCo**
This is in preparation for allowing module specs to be used here as well as ordinary pathnames.  Also renames "bundlePath" to "bundleSpec" for the same reasons.

### CODE REVIEW

File: packages/SwingSet/src/controller.js [hunk 1]
```diff
@@ -397,22 +397,22 @@ export async function buildVatController(
         validateBundleDescriptor(desc, groupName, name);
         if (!desc.bundleName) {
           names.push(name);
-          if (desc.sourcePath) {
-            const sourcePath = desc.sourcePath;
-            if (!(sourcePath[0] === '.' || path.isAbsolute(sourcePath))) {
+          if (desc.sourceSpec) {
+            const sourceSpec = desc.sourceSpec;
+            if (!(sourceSpec[0] === '.' || path.isAbsolute(sourceSpec))) {
```
Comment (michaelfig) [inline packages/SwingSet/src/controller.js:402]: This relative test would pass `.dotfile/boo`, which isn't a relative specifier, it's bare.

File: packages/SwingSet/src/controller.js [hunk 2]
```diff
@@ -397,22 +397,22 @@ export async function buildVatController(
...
               );
             }
-            presumptiveBundles.push(bundleSource(sourcePath));
-          } else if (desc.bundlePath) {
-            const bundlePath = desc.bundlePath;
-            if (!(bundlePath[0] === '.' || path.isAbsolute(bundlePath))) {
+            presumptiveBundles.push(bundleSource(sourceSpec));
+          } else if (desc.bundleSpec) {
+            const bundleSpec = desc.bundleSpec;
+            if (!(bundleSpec[0] === '.' || path.isAbsolute(bundleSpec))) {
```
Comment (michaelfig) [inline packages/SwingSet/src/controller.js:410]: Same.

**Review by michaelfig — APPROVED** [review]
LGTM.  You may end up fixing the relative path checking problem I noticed in a later PR, so this should not block the current PR if you want to land.

File: packages/SwingSet/src/controller.js (hunk 1 above)
Comment (FUDCo) [reply packages/SwingSet/src/controller.js:402]: Good catch.  That bit was cribbed from the code that existed prior to my overhaul of the config subsystem and I sort of blindly copied it without considering it too deeply.

Comment (FUDCo) [reply packages/SwingSet/src/controller.js:402]: Upon further consideration, it's not clear why we should even care in this context.  I will ping Brian.

**Pull request merged**